### DISCUSSION TITLE
Open message handler

### DISCUSSION THREAD

**Discussion by ksielyov** [invertase/react-native-firebase#4823] [Q&A] [answered]
Hello

I use @react-native-firebase/messaging. I want to handle the opening message, to show another content for user.

**Comment by JoeToeniskoetter** [comment, marked as answer]
If you are trying to handle a user clicking the notification itself 

```typescript
import messaging from '@react-native-firebase/messaging';


messaging().onNotificationOpenedApp(remoteMessage => {
      //navigate to new screen or
     //Show the user something
    });
```

[https://rnfirebase.io/messaging/notifications](url)

**Reply by ksielyov** [reply]
Thanks, I will try it
